chainio/clients/fireblocks/client.go

```diff
@@ -0,0 +1,168 @@
+package fireblocks
+
+import (
+	"bytes"
+	"context"
+	"crypto/rsa"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/golang-jwt/jwt"
+	"github.com/google/uuid"
+)
+
+type FireblocksTxID string
+
+type FireblocksClient interface {
+	// ContractCall makes a ContractCall request to the Fireblocks API.
+	// It signs and broadcasts a transaction and returns the transaction ID and status.
+	// ref: https://developers.fireblocks.com/reference/post_transactions
+	ContractCall(ctx context.Context, body *ContractCallRequest) (*ContractCallResponse, error)
+	// ListContracts makes a ListContracts request to the Fireblocks API
+	// It returns a list of whitelisted contracts and their assets for the account. This call is necessary to get the contract ID for a whitelisted contract.
```

Comment:
is the contract ID the `sourceaccountId` that is passed to `NewContractCallRequest`? If so can you just change the sentence to
```
This call is used to get the contract ID for a whitelisted contract, which is needed by NewContractCallRequest before making a ContractCall
```
or somethng like this?